app/controllers/api/v1/admin/site_settings_controller.rb

```diff
@@ -26,6 +26,7 @@ def update
 
           update = if params[:name] == 'BrandingImage'
                      site_setting.image.attach params[:site_setting][:value]
+                     site_setting.update(value: site_setting.image.blob.filename.to_s)
```

Comment:
Do we still need to store the file name?

Reply:
Nope, forgot to remove it from backend. Good catch